### PR TITLE
Updated smart and feature home, section polls

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@miltontony @saeedmb8 please review
screen shot 2016-07-14 at 5 10 33 pm

**Comment by Mitso** [comment]
Section Poll
screen shot 2016-07-14 at 5 11 56 pm

Feature Poll

screen shot 2016-07-14 at 5 13 11 pm

File: iogt/templates/registration/password_reset_done.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <div class="profiles">
 	<p>{% trans "We've emailed you instructions for setting your password, if an account exists with the email you entered. You should receive them shortly." %}</p>
 	<p>{% trans "If you don't receive an email, please make sure you've entered the address you registered with, and check your spam folder." %}</p>
-	<a href="{% url 'home' %}" class="call-to-action__nav-item call-to-action__nav-item--search-home">
+	<a href="{% url 'home' %}" class="call-to-action__nav-item call-to-action__nav-item--back-home">
```
Comment (smarzban) [inline iogt/templates/registration/password_reset_done.html]: @Mitso can you please change `{% url 'home' %}` to `{{request.site.root_page.url}}`

File: iogt/templates/search/search_results.html [hunk 2]
```diff
@@ -53,7 +53,7 @@ <h2 class="heading heading--search">{% trans "Search" %}</h2>
 		<h3 class="search__feedback-title">
 			{% trans "No search results for " %} {{search_query}}
 		</h3>
-		<a href="" class="call-to-action__nav-item call-to-action__nav-item--search-home">
+		<a href="/" class="call-to-action__nav-item call-to-action__nav-item--back-home">
```
Comment (smarzban) [inline iogt/templates/search/search_results.html]: @Mitso also here `/` to `{{request.site.root_page.url}}`

**Comment by smarzban** [comment]
@Mitso left small comments otherwise 👍  🍰

**Comment by Mitso** [comment]
Awesome, thanks Saeed :)

**Pull request merged**